airflow/providers/google/cloud/operators/bigquery.py

```diff
@@ -2068,6 +2069,7 @@ class BigQueryInsertJobOperator(BaseOperator):
         "configuration",
         "job_id",
         "impersonation_chain",
+        'project_id'
```

Comment:
```suggestion
        "project_id",
```

These will probably fix the static checks